guides/v2.3/graphql/mutations/set-guest-email.md

```diff
@@ -68,3 +68,14 @@ Attribute |  Data Type | Description
 {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find a cart with ID "XXX"` | Cart ID specified in the `cart` attribute does not exist.
+`Invalid email format` | Specified value in the `email` attribute has incorrect format.
+`Required parameter "cart_id" is missing` | Value in the `cart` attribute is empty or `cart` attribute did not specify in the mutation.
+`Required parameter "email" is missing` | Mutation `email` attribute has an empty value.
+`The current user cannot perform operations on cart "XXX"` | Not authorized user (guest) tried to set an email to customer's cart.
+`The request is not allowed for logged in customers` | Authorized user (customer) is unable to use `setGuestEmailOnCart` mutation.
```

Comment:
I would say "Authorized user (customer) is not allowed ...". Since "unable" here sounds like "not able for some reason". Thank you